tests/suites/helpers.function

```diff
@@ -175,6 +175,7 @@ static int redirect_output( FILE** out_stream, const char* path )
 
     if( *out_stream == NULL )
     {
+        close( stdout_fd );
```

Comment:
Could you please explain, why this `fd` should be closed here, when we have a `fclose` call earlier? It seems that the file associated with this file descriptor has already been closed, but I may be missing some nuance here.

Reply:
I think the fclose() call above closes *out_stream, but not the file descriptor in stdout_fd that is a copy of *out_stream created using dup() in line 165. Take a look at the internal reference IOTSSL-2624 for information on the issue reported by coverity.

Reply:
Thanks for the answer. I can now see, that, the FD is duped, and therefore needs to be closed separately.